README.md

```diff
@@ -12,17 +12,22 @@ Playbook is the first design system built for both Rails & React interfaces. Ins
 - [asdf](https://github.com/asdf-vm/asdf)
 - Install language tools: `asdf install`
 
-### Getting started
-
-1. Install commit hooks `yarn prepare`
-1. Install all projects dependencies: `yarn install`
-1. Install website ruby dependencies: `cd playbook-website && bundle`
-  1. If you receive a bundle(r) related error, be sure that `bundler` is installed first. See the `BUNDLED WITH` section of `playbook-website/Gemfile.lock` for the exact version to install.
-
-### Running the website for development
-
-1. Start watchers and servers: `yarn start-dev`
-1. Open [http://localhost:3000](http://localhost:3000)
+### Getting Started and Running Playbook for Development
+
+1. After cloning the repo, you should have the following nested folders among other files. It will be important to pay attention to which folder we are in as we get playbook running:
+    ```
+    playbook
+    │   playbook
+    │   playbook-website
+    ```
+2. From the top-level playbook folder run: `yarn prepare`
+3. Again from the top-level playbook folder run: `yarn install`
+4. cd into the playbook-website folder: `cd playbook-website`
+5. From the playbook-website folder run: `bundle install`
```

Comment:
```suggestion
3. cd into the playbook-website folder: `cd playbook-website && bundle install`
```

Reply:
I think this one needs to remain separate because it's moving folders and running bundle and those are two different types of commands. If this is for beginners as well as nitro developers, it's good to keep those separate.